### PR TITLE
fix scrollTo when canvas is scaled

### PR DESCRIPTION
**Pull request by BrianHung**

### CODE REVIEW

**Comment by jassmith** [comment]
Failing test :)

File: packages/core/src/data-editor/data-editor.tsx [hunk 1]
```diff
@@ -1367,6 +1367,7 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
                     }
 
                     const scrollBounds = canvas.getBoundingClientRect();
+                    const scale = scrollBounds.width / canvas.offsetWidth;
```
Comment (jassmith) [inline packages/core/src/data-editor/data-editor.tsx:1370]: Does this work right when there is a physical scrollbar?

Comment (BrianHung) [reply packages/core/src/data-editor/data-editor.tsx:1370]: What do you mean by physical scroll bar?

Does physical scroll bar width affect canvas width and scale somehow?

Comment (jassmith) [reply packages/core/src/data-editor/data-editor.tsx:1370]: I think if there is a scrollbar (like not an overlay one, a permanent one) the scrollBounds and the canvas bounds will have different widths when unscaled.

Comment (BrianHung) [reply packages/core/src/data-editor/data-editor.tsx:1370]: Testing this on a windows computer, the scaling factor computed above is almost exact to what I set via transform (some trailing decimal points that are minor).

Comment (jassmith) [reply packages/core/src/data-editor/data-editor.tsx:1370]: 👍 will test myself thank you

**Pull request merged**